css/main.css

```diff
@@ -51,6 +71,22 @@ header {
   color: white;
 }
 
+.menu-bars i.fa-times {
```

Comment:
**Kindly stick to the Figma design as possible** 

Figma template :
![image](https://user-images.githubusercontent.com/87448628/157621518-da2cff27-6cb0-4975-bd15-ba1cc927b0c8.png)

Your Desgin:

![image](https://user-images.githubusercontent.com/87448628/157621620-97a2f84b-92da-46b5-be09-63ec5dbeb5dc.png)